index.js

```diff
@@ -35,7 +35,7 @@ module.exports = function (headerText, data) {
     var template = data === false ? headerText : gutil.template(headerText, extend({ file: file, filename: filename }, data));
     concat = new Concat(true, filename);
 
-    if (fs.lstatSync(file.path).isDirectory()) {
+    if (fs.existsSync(file.path) && fs.lstatSync(file.path).isDirectory()) {
```

Comment:
Please do not use fs.exist(Sync). See https://github.com/tracker1/gulp-header/issues/36#issuecomment-226736047